File: app/src/main/java/com/nhq/authenticator/data/repository/remote/TimeRetrofitService.java
package com.nhq.authenticator.data.repository.remote;

import com.nhq.authenticator.data.entity.ResponseTime;

import retrofit2.Call;
import retrofit2.http.GET;

public interface TimeRetrofitService {
    @GET("api/ip")
    Call<ResponseTime> getTimeViaPublicIp();
}

File: app/src/main/java/com/nhq/authenticator/data/repository/remote/TimeRetrofitService.kt
package com.nhq.authenticator.data.repository.remote

import com.nhq.authenticator.data.entity.ResponseTime
import retrofit2.Call
import retrofit2.http.GET

interface TimeRetrofitService {
    @get:GET("api/ip")
    val timeViaPublicIp: Call<ResponseTime?>
}